node/src/store.rs

```diff
@@ -148,16 +146,18 @@ pub trait Store: Send + Sync + Debug {
 
     /// Insert a range of headers into the store.
     ///
-    /// Inserts are allowed at the front of the store or at the ends of any existing ranges. Edges
-    /// of inserted header ranges are verified against headers present in the store, if they
-    /// exist.
+    /// New insertion should pass all the constrains in [`BlockRanges::check_insertion_constrains`],
```

Comment:
```suggestion
    /// New insertion should pass all the constraints in [`BlockRanges::check_insertion_constraints`],
```
constrain = verb, constraint = noun